content/organization/tech/security/index.en.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Security"
+menuTitle: "Security"
+weight: 54
+---
+
+As organizers you will be responsible for accounts such as email, social media accounts.
+Here are some steps that will help reduce the risk of someone's hacking into your account.
+
+## Use a personal password manager!
+
+For your personal use (for R-Ladies things and in general!), we strongly recommend using a personal password manager, be it a free one like KeePass (with a back-up you'd set up on Dropbox or Google Drive) or a paid service like 1Password.
+
+If you feel on the fence, ask other organizers or your friends about their experience with some password managers.
+
+### KeePass workflow
+
+* save all your passwords in a KeePass database.
+* have it backed-up on Dropbox or Google Drive.
+* only learn two passwords: the one to decrypt the KeePass database, the one to your Google Drive account.
+
```

Comment:
### Bitwarden workflow

Free for single person, or two person 'organisation' (ie. ability share certain passwords with a single other person). 
Download the Desktop client and/or the extension/plugin for your browser of choice. 
Choose a password you can remember for Bitwarden, and store all your passwords in there. 
Also has biometrics integration for those using it.